Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py

```diff
@@ -400,6 +425,18 @@ def get_last_mirrored_in_time(args):
     return last_mirrored_in_timestamp
 
 
+def get_multiple_incidents_extra_data_response(client, args) -> bool:
```

Comment:
Please add docstring describing the method, consider also to change the name to more accurate and clear name,